### PR TITLE
[CHAT-1412]: Add Thai to `Empty` sample

### PR DESCRIPTION
**Pull request by mbektchiev**
refs https://kinvey.atlassian.net/browse/CHAT-1412

### CODE REVIEW

File: Empty/th/cognitive-flow.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      "ฉันกำลังกลับไปที่ {{ conversationDisplayName }}"
+    ]
+  },
+  "commands": {
+    "NEXT-PAGE": [
+      "ถัดไป 5"
+    ],
+    "RESTART": [
+      "เริ่มต้นใหม่",
+      "restart"
```
Comment (Mitko-Kerezov) [inline Empty/th/cognitive-flow.json:67]: Is this left on purpose?

File: Empty/th/training.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            "แล้วพบกันใหม่",
+            "ลาก่อน"
+          ]
+        },
+        {
+          "value": "thanks",
+          "expressions": [
+            "ขอบคุณ",
+            "ขอบคุณเช่นกัน",
+            "ขอบคุณ bot",
```
Comment (Mitko-Kerezov) [inline Empty/th/training.json:77]:
```suggestion
            "ขอบคุณบอท",
```

**Review by Mitko-Kerezov — CHANGES_REQUESTED** [review]

File: Empty/th/cognitive-flow.json (hunk 1 above)
Comment (mbektchiev) [reply Empty/th/cognitive-flow.json:67]: Yes, I saw that in the Bulgarian sample we have `restart` in English as well, and decided that it's not a bad idea

File: Empty/th/training.json (hunk 2 above)
Comment (mbektchiev) [reply Empty/th/training.json:77]: image

Comment (Mitko-Kerezov) [reply Empty/th/training.json:77]: unsure, however I am fairly certain as there is a native Thai word for "bot":
image

I'm just raising awareness here - can't really be sure without knowledge in the language 😞

**Review by Mitko-Kerezov — APPROVED** [review]

**Pull request merged**